workspaces/ai-integrations/plugins/catalog-backend-module-model-catalog/src/clients/ModelCatalogGenerator.ts

```diff
@@ -24,10 +24,12 @@ import {
   ComponentEntity,
   ApiEntity,
   ResourceEntity,
+  makeValidator,
 } from '@backstage/catalog-model';
+import { LoggerService } from '@backstage/backend-plugin-api';
 
 function isModelCatalog(o: any): o is ModelCatalog {
-  return 'model' in o && 'modelServer' in o;
+  return 'models' in o || 'modelServers' in o;
```

Comment:
was going from singular to plural intentional ?

Reply:
Updated

Reply:
looks good